matplotlibrc.template

```diff
@@ -154,447 +154,463 @@ backend      : $TEMPLATE_BACKEND
 
 ###ANIMATION settings
-#animation.html :  none            # How to display the animation as HTML in
-                                   # the IPython notebook. 'html5' uses
-                                   # HTML5 video tag; 'jshtml' creates a 
-								   # Javascript animation
-#animation.writer : ffmpeg         # MovieWriter 'backend' to use
-#animation.codec : h264            # Codec to use for writing movie
-#animation.bitrate: -1             # Controls size/quality tradeoff for movie.
-                                   # -1 implies let utility auto-determine
-#animation.frame_format:  png      # Controls frame format used by temp files
-#animation.html_args: ''           # Additional arguments to pass to html writer
-#animation.ffmpeg_path: 'ffmpeg'   # Path to ffmpeg binary. Without full path
-                                   # $PATH is searched
-#animation.ffmpeg_args: ''         # Additional arguments to pass to ffmpeg
-#animation.avconv_path: 'avconv'   # Path to avconv binary. Without full path
-                                   # $PATH is searched
-#animation.avconv_args: ''         # Additional arguments to pass to avconv
-#animation.convert_path: 'convert' # Path to ImageMagick's convert binary.
-                                   # On Windows use the full path since convert
-                                   # is also the name of a system tool.
-#animation.convert_args: ''        # Additional arguments to pass to convert
+#animation.html :  none            ## How to display the animation as HTML in
+                                   ## the IPython notebook. 'html5' uses
+                                   ## HTML5 video tag; 'jshtml' creates a 
+                                   ## Javascript animation
+#animation.writer : ffmpeg         ## MovieWriter 'backend' to use
+#animation.codec : h264            ## Codec to use for writing movie
+#animation.bitrate: -1             ## Controls size/quality tradeoff for movie.
+                                   ## -1 implies let utility auto-determine
+#animation.frame_format:  png      ## Controls frame format used by temp files
+#animation.html_args: ''           ## Additional arguments to pass to html writer
+#animation.ffmpeg_path: 'ffmpeg'   ## Path to ffmpeg binary. Without full path
+                                   ## $PATH is searched
+#animation.ffmpeg_args: ''         ## Additional arguments to pass to ffmpeg
+#animation.avconv_path: 'avconv'   ## Path to avconv binary. Without full path
+                                   ## $PATH is searched
+#animation.avconv_args: ''         ## Additional arguments to pass to avconv
+#animation.convert_path: 'convert' ## Path to ImageMagick's convert binary.
+                                   ## On Windows use the full path since convert
+                                   ## is also the name of a system tool.
+#animation.convert_args: ''        ## Additional arguments to pass to convert
```

Comment:
Here and in all cases above, there should be no quotes for strings.  A blank field is an empty string.